test/integration/createCoursesFile.test.js

```diff
@@ -1,18 +1,24 @@
 require('dotenv').config()
 const test = require('tape')
 const fs = require('fs-extra')
+const rewire = require('rewire')
```

Comment:
Why do we require `rewire` now instead of a simple `require` to get the needed functionality?

Reply:
Is has been made "private" :)

Reply:
The needed functionality was removed from module export.